fitlins/cli/run.py

```diff
@@ -88,10 +88,10 @@ def get_parser():
                         choices=['MNI152NLin2009cAsym', ''],
                         default='MNI152NLin2009cAsym',
                         help='registered space of input datasets. Empty value for no explicit space.')
-    g_bids.add_argument('--include', action='store', default=None,
+    g_bids.add_argument('--force-index', action='store', default=None,
                         help='regex pattern to include files')
```

Comment:
These are no longer regexes, right? Just paths?

Reply:
Actually, they're both regexes so we should apply the same logic

Reply:
You are a strong contender for code reviewer of the year 🏅

Reply:
You get an emoji medal. Treasure it.